api/management/commands/loaddata.py

```diff
@@ -0,0 +1,17 @@
+import os
+import csv
+from django.core.management.base import BaseCommand
+from api.models import Disease
+
+
+class Command(BaseCommand):
```

Comment:
Style: Blank line should follow. See [PEP8 for blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines).